geps/gep-1897.md

```diff
@@ -25,11 +25,17 @@ Gateway to backend**, such as TLS versions and cipher suites. (Use case #5 in [G
 
 ## Longer Term Goals
 
-These are worthy goals, but may need a different GEP for proper attention.
+These are worthy goals, but may need a different GEP for proper attention.  This GEP is concerned entirely with the
+controlplane, i.e. the hop between gateway and backend.
 
-1. TCPRoute use cases (completed by GA)
-2. Mutual TLS use cases
-3. Service mesh use cases
+1. [TCPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.TCPRoute) and
+[GRPCRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.GRPCRoute) use cases
+are not addressed here, because at this point in time these two route types are not graduated to beta.
+2. Mutual TLS (mTLS) use cases are not thoroughly addressed here because we have tried and failed in the past to
+generalize around mTLS, and the treatment of mTLS should be taken outside the context of the hop between gateway and
+backend.
```

Comment:
Okay, I will incorporate this.

Reply:
Actually, based on the commentary from Nick, I'm going to use this.  It doesn't include what might need to happen in the future for mTLS.  Sound okay?

Mutual TLS (mTLS) use cases are intentionally out of scope for this GEP because this GEP does not address the case where connections to TLS are implicitly configured on behalf of the user, such as in Service Mesh.  It is about the case where an application developer needs to explicitly express that they expect TLS when there is no automatic, implicit configuration of that.

Reply:
Was updated again later.